### ISSUE TITLE
Bug: VyprVPN OpenVPN fails because of compression

### ISSUE THREAD

**Issue by D43m0n** [qdm12/gluetun#671]
### Is this urgent?

No

### Host OS

docker

### CPU arch

x86_64

### VPN service provider

VyprVPN

### What are you using to run the container

docker-compose

### What is the version of Gluetun

Running version v3.25.0 built on 2021-09-28T14:27:44Z (commit 62a6016)

### What's the problem 🤔

Gluetun isn't able to create a working VPN connection anymore to the Internet via OpenVPN and my provider (VyprVPN).  My containers that use the gluetun container are accessible from the local LAN where docker resides, but the gluetun container can't sustain a VPN connection anymore.

### Share your logs

```log
2021/10/14 16:49:32 INFO Alpine version: 3.14.2
 2021/10/14 16:49:32 INFO OpenVPN 2.4 version: 2.4.11
 2021/10/14 16:49:32 INFO OpenVPN 2.5 version: 2.5.2
 2021/10/14 16:49:32 INFO Unbound version: 1.13.2
 2021/10/14 16:49:32 INFO IPtables version: v1.8.7
 2021/10/14 16:49:32 INFO Settings summary below:
 |--VPN:
    |--Type: openvpn
    |--OpenVPN:
       |--Version: 2.5
       |--Verbosity level: 1
       |--Network interface: tun0
       |--Run as root: enabled
    |--Vyprvpn settings:
       |--Regions: netherlands
       |--OpenVPN selection:
          |--Protocol: udp
 |--DNS:
    |--Plaintext address: 1.1.1.1
    |--DNS over TLS:
       |--Unbound:
           |--DNS over TLS providers:
               |--Cloudflare
           |--Listening port: 53
           |--Access control:
               |--Allowed:
                   |--0.0.0.0/0
                   |--::/0
           |--Caching: enabled
           |--IPv4 resolution: enabled
           |--IPv6 resolution: disabled
           |--Verbosity level: 1/5
           |--Verbosity details level: 0/4
           |--Validation log level: 0/2
           |--Username: 
       |--Blacklist:
          |--Blocked categories: malicious
          |--Additional IP networks blocked: 13
       |--Update: every 24h0m0s
 |--Firewall:
 |--Log:
    |--Level: INFO
 |--System:
    |--Process user ID: 1000
    |--Process group ID: 1000
    |--Timezone: europe/amsterdam
 |--Health:
    |--Server address: 127.0.0.1:9999
    |--Address to ping: github.com                                                                                                                                                                                                [340/1837]
    |--VPN:
       |--Initial duration: 6s
       |--Addition duration: 5s
 |--HTTP control server:
    |--Listening port: 8000
    |--Logging: enabled
 |--Public IP getter:
    |--Fetch period: 12h0m0s
    |--IP file: /tmp/gluetun/ip
 |--Github version information: enabled
 2021/10/14 16:49:32 INFO routing: default route found: interface eth0, gateway 172.17.0.1
 2021/10/14 16:49:32 INFO routing: local ethernet link found: eth0
 2021/10/14 16:49:32 INFO routing: local ipnet found: 172.17.0.0/16
 2021/10/14 16:49:32 INFO routing: default route found: interface eth0, gateway 172.17.0.1
 2021/10/14 16:49:32 INFO routing: adding route for 0.0.0.0/0
 2021/10/14 16:49:32 INFO firewall: firewall disabled, only updating allowed subnets internal list
 2021/10/14 16:49:32 INFO routing: default route found: interface eth0, gateway 172.17.0.1
 2021/10/14 16:49:32 INFO TUN device is not available: open /dev/net/tun: no such file or directory; creating it...
 2021/10/14 16:49:32 INFO firewall: enabling...
 2021/10/14 16:49:32 INFO firewall: enabled successfully
 2021/10/14 16:49:32 INFO healthcheck: listening on 127.0.0.1:9999
 2021/10/14 16:49:32 INFO http server: listening on :8000
 2021/10/14 16:49:32 INFO dns over tls: using plaintext DNS at address 1.1.1.1
 2021/10/14 16:49:32 INFO firewall: setting VPN connection through firewall...
 2021/10/14 16:49:32 INFO openvpn: OpenVPN 2.5.2 x86_64-alpine-linux-musl [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [MH/PKTINFO] [AEAD] built on May  4 2021
 2021/10/14 16:49:32 INFO openvpn: library versions: OpenSSL 1.1.1l  24 Aug 2021, LZO 2.10
 2021/10/14 16:49:32 INFO openvpn: TCP/UDP: Preserving recently used remote address: [AF_INET]128.90.96.16:443
 2021/10/14 16:49:32 INFO openvpn: UDP link local: (not bound)
 2021/10/14 16:49:32 INFO openvpn: UDP link remote: [AF_INET]128.90.96.16:443
 2021/10/14 16:49:32 WARN openvpn: 'link-mtu' is used inconsistently, local='link-mtu 1569', remote='link-mtu 1570'
 2021/10/14 16:49:32 WARN openvpn: 'comp-lzo' is present in remote config but missing in local config, remote='comp-lzo'
 2021/10/14 16:49:32 INFO openvpn: [eu1.vyprvpn.com] Peer Connection Initiated with [AF_INET]128.90.96.16:443
 2021/10/14 16:49:33 INFO openvpn: TUN/TAP device tun0 opened
 2021/10/14 16:49:33 INFO openvpn: /sbin/ip link set dev tun0 up mtu 1500
 2021/10/14 16:49:33 INFO openvpn: /sbin/ip link set dev tun0 up
 2021/10/14 16:49:33 INFO openvpn: /sbin/ip addr add dev tun0 10.2.2.20/24
 2021/10/14 16:49:33 INFO openvpn: Initialization Sequence Completed
 2021/10/14 16:49:33 INFO dns over tls: downloading DNS over TLS cryptographic files
 2021/10/14 16:49:43 INFO openvpn: write to TUN/TAP : Invalid argument (code=22)
 2021/10/14 16:49:48 WARN dns over tls: cannot update files 
 2021/10/14 16:49:48 INFO dns over tls: attempting restart in 10s
 2021/10/14 16:49:53 INFO openvpn: write to TUN/TAP : Invalid argument (code=22)
 2021/10/14 16:49:58 INFO dns over tls: downloading DNS over TLS cryptographic files
 2021/10/14 16:50:03 INFO openvpn: write to TUN/TAP : Invalid argument (code=22)
 2021/10/14 16:50:03 ERROR ip getter: Get "http://ip1.dynupdate.no-ip.com:8245": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
 2021/10/14 16:50:03 INFO ip getter: retrying in 5s
 2021/10/14 16:50:03 ERROR vpn: cannot get version information: Get "https://api.github.com/repos/qdm12/gluetun/releases": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
 2021/10/14 16:50:04 INFO healthcheck: program has been unhealthy for 6s: restarting VPN
 2021/10/14 16:50:04 INFO vpn: stopping
 2021/10/14 16:50:04 INFO vpn: starting
 2021/10/14 16:50:04 INFO firewall: setting VPN connection through firewall...
 2021/10/14 16:50:04 INFO openvpn: OpenVPN 2.5.2 x86_64-alpine-linux-musl [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [MH/PKTINFO] [AEAD] built on May  4 2021
 2021/10/14 16:50:04 INFO openvpn: library versions: OpenSSL 1.1.1l  24 Aug 2021, LZO 2.10
 2021/10/14 16:50:04 INFO openvpn: TCP/UDP: Preserving recently used remote address: [AF_INET]128.90.96.16:443
 2021/10/14 16:50:04 INFO openvpn: UDP link local: (not bound)
 2021/10/14 16:50:04 INFO openvpn: UDP link remote: [AF_INET]128.90.96.16:443
 2021/10/14 16:50:04 WARN openvpn: 'link-mtu' is used inconsistently, local='link-mtu 1569', remote='link-mtu 1570'
 2021/10/14 16:50:04 WARN openvpn: 'comp-lzo' is present in remote config but missing in local config, remote='comp-lzo'
 2021/10/14 16:50:04 INFO openvpn: [eu1.vyprvpn.com] Peer Connection Initiated with [AF_INET]128.90.96.16:443
 2021/10/14 16:50:05 INFO openvpn: TUN/TAP device tun0 opened
 2021/10/14 16:50:05 INFO openvpn: /sbin/ip link set dev tun0 up mtu 1500
2021/10/14 16:50:05 INFO openvpn: /sbin/ip link set dev tun0 up                                                                                                                                                                  [278/1837]
 2021/10/14 16:50:05 INFO openvpn: /sbin/ip addr add dev tun0 10.2.2.22/24
 2021/10/14 16:50:05 ERROR openvpn: RTNETLINK answers: File exists
 2021/10/14 16:50:05 ERROR openvpn: Linux route add command failed: external program exited with error status: 2
 2021/10/14 16:50:05 INFO openvpn: Initialization Sequence Completed
 2021/10/14 16:50:13 WARN dns over tls: cannot update files 
 2021/10/14 16:50:13 INFO dns over tls: attempting restart in 20s
 2021/10/14 16:50:16 INFO openvpn: write to TUN/TAP : Invalid argument (code=22)
 2021/10/14 16:50:23 ERROR ip getter: Get "https://ipinfo.io/ip": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
 2021/10/14 16:50:23 INFO ip getter: retrying in 5s
 2021/10/14 16:50:26 INFO openvpn: write to TUN/TAP : Invalid argument (code=22)
 2021/10/14 16:50:33 INFO dns over tls: downloading DNS over TLS cryptographic files
 2021/10/14 16:50:36 INFO healthcheck: program has been unhealthy for 11s: restarting VPN
 2021/10/14 16:50:36 INFO vpn: stopping
 2021/10/14 16:50:36 INFO vpn: starting
 2021/10/14 16:50:36 INFO firewall: setting VPN connection through firewall...
 2021/10/14 16:50:36 INFO openvpn: OpenVPN 2.5.2 x86_64-alpine-linux-musl [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [MH/PKTINFO] [AEAD] built on May  4 2021
 2021/10/14 16:50:36 INFO openvpn: library versions: OpenSSL 1.1.1l  24 Aug 2021, LZO 2.10
 2021/10/14 16:50:36 INFO openvpn: TCP/UDP: Preserving recently used remote address: [AF_INET]128.90.96.16:443
 2021/10/14 16:50:36 INFO openvpn: UDP link local: (not bound)
 2021/10/14 16:50:36 INFO openvpn: UDP link remote: [AF_INET]128.90.96.16:443
 2021/10/14 16:50:36 WARN openvpn: 'link-mtu' is used inconsistently, local='link-mtu 1569', remote='link-mtu 1570'
 2021/10/14 16:50:36 WARN openvpn: 'comp-lzo' is present in remote config but missing in local config, remote='comp-lzo'
 2021/10/14 16:50:36 INFO openvpn: [eu1.vyprvpn.com] Peer Connection Initiated with [AF_INET]128.90.96.16:443
 2021/10/14 16:50:37 INFO openvpn: TUN/TAP device tun0 opened
 2021/10/14 16:50:37 INFO openvpn: /sbin/ip link set dev tun0 up mtu 1500
 2021/10/14 16:50:37 INFO openvpn: /sbin/ip link set dev tun0 up
 2021/10/14 16:50:37 INFO openvpn: /sbin/ip addr add dev tun0 10.2.2.241/24
 2021/10/14 16:50:37 ERROR openvpn: RTNETLINK answers: File exists
 2021/10/14 16:50:37 ERROR openvpn: Linux route add command failed: external program exited with error status: 2
 2021/10/14 16:50:37 INFO openvpn: Initialization Sequence Completed
 2021/10/14 16:50:43 ERROR ip getter: Get "https://ipinfo.io/ip": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
 2021/10/14 16:50:43 INFO ip getter: retrying in 5s
 2021/10/14 16:50:47 INFO openvpn: write to TUN/TAP : Invalid argument (code=22)
 2021/10/14 16:50:48 WARN dns over tls: cannot update files 
 2021/10/14 16:50:48 INFO dns over tls: attempting restart in 40s
 2021/10/14 16:50:57 INFO openvpn: write to TUN/TAP : Invalid argument (code=22)
 2021/10/14 16:51:03 ERROR ip getter: Get "https://api.ipify.org": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
 2021/10/14 16:51:03 INFO ip getter: retrying in 5s
 2021/10/14 16:51:07 INFO openvpn: write to TUN/TAP : Invalid argument (code=22)
 2021/10/14 16:51:08 INFO healthcheck: program has been unhealthy for 16s: restarting VPN
 2021/10/14 16:51:08 INFO vpn: stopping
 2021/10/14 16:51:08 INFO vpn: starting
 2021/10/14 16:51:08 INFO firewall: setting VPN connection through firewall...
 2021/10/14 16:51:08 INFO openvpn: OpenVPN 2.5.2 x86_64-alpine-linux-musl [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [MH/PKTINFO] [AEAD] built on May  4 2021
 2021/10/14 16:51:08 INFO openvpn: library versions: OpenSSL 1.1.1l  24 Aug 2021, LZO 2.10
 2021/10/14 16:51:08 INFO openvpn: TCP/UDP: Preserving recently used remote address: [AF_INET]128.90.96.16:443
 2021/10/14 16:51:08 INFO openvpn: UDP link local: (not bound)
 2021/10/14 16:51:08 INFO openvpn: UDP link remote: [AF_INET]128.90.96.16:443
 2021/10/14 16:51:08 ERROR ip getter: Get "http://ip1.dynupdate.no-ip.com:8245": dial tcp: lookup ip1.dynupdate.no-ip.com on 1.1.1.1:53: write udp 172.17.0.2:42627->1.1.1.1:53: write: operation not permitted
 2021/10/14 16:51:08 INFO ip getter: retrying in 5s
 2021/10/14 16:51:09 WARN openvpn: 'link-mtu' is used inconsistently, local='link-mtu 1569', remote='link-mtu 1570'
 2021/10/14 16:51:09 WARN openvpn: 'comp-lzo' is present in remote config but missing in local config, remote='comp-lzo'
 2021/10/14 16:51:09 INFO openvpn: [eu1.vyprvpn.com] Peer Connection Initiated with [AF_INET]128.90.96.16:443
 2021/10/14 16:51:10 INFO openvpn: TUN/TAP device tun0 opened
 2021/10/14 16:51:10 INFO openvpn: /sbin/ip link set dev tun0 up mtu 1500
 2021/10/14 16:51:10 INFO openvpn: /sbin/ip link set dev tun0 up
 2021/10/14 16:51:10 INFO openvpn: /sbin/ip addr add dev tun0 10.2.2.252/24
 2021/10/14 16:51:10 ERROR openvpn: RTNETLINK answers: File exists
 2021/10/14 16:51:10 ERROR openvpn: Linux route add command failed: external program exited with error status: 2
 2021/10/14 16:51:10 INFO openvpn: Initialization Sequence Completed
 2021/10/14 16:51:21 INFO openvpn: write to TUN/TAP : Invalid argument (code=22)
2021/10/14 16:51:28 INFO dns over tls: downloading DNS over TLS cryptographic files
 2021/10/14 16:51:28 ERROR ip getter: Get "https://ifconfig.io/ip": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
 2021/10/14 16:51:28 INFO ip getter: retrying in 5s
 2021/10/14 16:51:31 INFO openvpn: write to TUN/TAP : Invalid argument (code=22)
 2021/10/14 16:51:40 INFO healthcheck: program has been unhealthy for 21s: restarting VPN
 2021/10/14 16:51:40 INFO vpn: stopping
 2021/10/14 16:51:40 INFO vpn: starting
 2021/10/14 16:51:40 INFO firewall: setting VPN connection through firewall...
 2021/10/14 16:51:40 INFO openvpn: OpenVPN 2.5.2 x86_64-alpine-linux-musl [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [MH/PKTINFO] [AEAD] built on May  4 2021
 2021/10/14 16:51:40 INFO openvpn: library versions: OpenSSL 1.1.1l  24 Aug 2021, LZO 2.10
 2021/10/14 16:51:40 INFO openvpn: TCP/UDP: Preserving recently used remote address: [AF_INET]128.90.96.16:443
 2021/10/14 16:51:40 INFO openvpn: UDP link local: (not bound)
 2021/10/14 16:51:40 INFO openvpn: UDP link remote: [AF_INET]128.90.96.16:443
 2021/10/14 16:51:41 WARN openvpn: 'link-mtu' is used inconsistently, local='link-mtu 1569', remote='link-mtu 1570'
 2021/10/14 16:51:41 WARN openvpn: 'comp-lzo' is present in remote config but missing in local config, remote='comp-lzo'
 2021/10/14 16:51:41 INFO openvpn: [eu1.vyprvpn.com] Peer Connection Initiated with [AF_INET]128.90.96.16:443
 2021/10/14 16:51:42 INFO openvpn: TUN/TAP device tun0 opened
 2021/10/14 16:51:42 INFO openvpn: /sbin/ip link set dev tun0 up mtu 1500
 2021/10/14 16:51:42 INFO openvpn: /sbin/ip link set dev tun0 up
 2021/10/14 16:51:42 INFO openvpn: /sbin/ip addr add dev tun0 10.2.2.199/24
 2021/10/14 16:51:42 ERROR openvpn: RTNETLINK answers: File exists
 2021/10/14 16:51:42 ERROR openvpn: Linux route add command failed: external program exited with error status: 2
 2021/10/14 16:51:42 INFO openvpn: Initialization Sequence Completed
 2021/10/14 16:51:43 WARN dns over tls: cannot update files 
 2021/10/14 16:51:43 INFO dns over tls: attempting restart in 1m20s
 2021/10/14 16:51:48 ERROR ip getter: Get "http://ip1.dynupdate.no-ip.com:8245": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
 2021/10/14 16:51:48 INFO ip getter: retrying in 5s
 2021/10/14 16:51:52 INFO openvpn: write to TUN/TAP : Invalid argument (code=22)
 2021/10/14 16:52:02 INFO openvpn: write to TUN/TAP : Invalid argument (code=22)
 2021/10/14 16:52:08 ERROR ip getter: Get "https://diagnostic.opendns.com/myip": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
 2021/10/14 16:52:08 INFO ip getter: retrying in 5s
 2021/10/14 16:52:12 INFO openvpn: write to TUN/TAP : Invalid argument (code=22)
 2021/10/14 16:52:12 INFO healthcheck: program has been unhealthy for 26s: restarting VPN
 2021/10/14 16:52:12 INFO vpn: stopping
 2021/10/14 16:52:12 INFO vpn: starting
 2021/10/14 16:52:12 INFO firewall: setting VPN connection through firewall...
 2021/10/14 16:52:12 INFO openvpn: OpenVPN 2.5.2 x86_64-alpine-linux-musl [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [MH/PKTINFO] [AEAD] built on May  4 2021
 2021/10/14 16:52:12 INFO openvpn: library versions: OpenSSL 1.1.1l  24 Aug 2021, LZO 2.10
 2021/10/14 16:52:12 INFO openvpn: TCP/UDP: Preserving recently used remote address: [AF_INET]128.90.96.16:443
 2021/10/14 16:52:12 INFO openvpn: UDP link local: (not bound)
 2021/10/14 16:52:12 INFO openvpn: UDP link remote: [AF_INET]128.90.96.16:443
 2021/10/14 16:52:13 WARN openvpn: 'link-mtu' is used inconsistently, local='link-mtu 1569', remote='link-mtu 1570'
 2021/10/14 16:52:13 WARN openvpn: 'comp-lzo' is present in remote config but missing in local config, remote='comp-lzo'
 2021/10/14 16:52:13 INFO openvpn: [eu1.vyprvpn.com] Peer Connection Initiated with [AF_INET]128.90.96.16:443
 2021/10/14 16:52:13 ERROR ip getter: Get "https://ifconfig.me/ip": dial tcp: lookup ifconfig.me on 1.1.1.1:53: write udp 172.17.0.2:42171->1.1.1.1:53: write: operation not permitted
 2021/10/14 16:52:13 INFO ip getter: retrying in 5s
 2021/10/14 16:52:14 INFO openvpn: TUN/TAP device tun0 opened
 2021/10/14 16:52:14 INFO openvpn: /sbin/ip link set dev tun0 up mtu 1500
 2021/10/14 16:52:14 INFO openvpn: /sbin/ip link set dev tun0 up
 2021/10/14 16:52:14 INFO openvpn: /sbin/ip addr add dev tun0 10.2.2.221/24
 2021/10/14 16:52:14 ERROR openvpn: RTNETLINK answers: File exists
 2021/10/14 16:52:14 ERROR openvpn: Linux route add command failed: external program exited with error status: 2
 2021/10/14 16:52:14 INFO openvpn: Initialization Sequence Completed
 2021/10/14 16:52:24 INFO openvpn: write to TUN/TAP : Invalid argument (code=22)
 2021/10/14 16:52:33 ERROR ip getter: Get "https://diagnostic.opendns.com/myip": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
 2021/10/14 16:52:33 INFO ip getter: retrying in 5s
 2021/10/14 16:52:34 INFO openvpn: write to TUN/TAP : Invalid argument (code=22)
 2021/10/14 16:52:44 INFO healthcheck: program has been unhealthy for 31s: restarting VPN
 2021/10/14 16:52:44 INFO vpn: stopping
 2021/10/14 16:52:44 INFO vpn: starting
```


### Share your configuration

```yml
gluetun:                                                                                                                                                                                                                                  
    image: qmcgaw/gluetun:v3.25.0                                                                                                                                                                                                           
    #image: qmcgaw/gluetun                                                                                                                                                                                                                  
    container_name: gluetun                                                                                                                                                                                                                 
    cap_add:                                                                                                                                                                                                                                
      - NET_ADMIN                                                                                                                                                                                                                           
    environment:                                                                                                                                                                                                                            
      - TZ=${TZ}                                                                                                                                                                                                                            
      - VPNSP=vyprvpn                                                                                                                                                                                                                       
      - OPENVPN_USER=${OPENVPN_USER}                                                                                                                                                                                                        
      - OPENVPN_PASSWORD=${OPENVPN_PASSWORD}                                                                                                                                                                                                
      - REGION=${OPENVPN_REGION}                                                                                                                                                                                                            
    volumes:                                                                                                                                                                                                                                
      - "${CONFIG_DIRECTORY}/gluetun:/gluetun"                                                                                                                                                                                              
    network_mode: bridge                                                                                                                                                                                                                    
    ports:                                                                                                                                                                                                                                  
      - 8888:8888/tcp # HTTP proxy                                                                                                                                                                                                          
      - 8388:8388/tcp # Shadowsocks                                                                                                                                                                                                         
      - 8388:8388/udp # Shadowsocks                                                                                                                                                                                                         
      - 8000:8000/tcp # Built-in HTTP control server                                                                                                                                                                                        
      - 8080:8080     # port to access SABnzbd                                                                                                                                                                                              
      - 5076:5076     # port to access nzbhydra                                                                                                                                                                                             
    restart: unless-stopped 

  sabnzbd:                                                                                                                                                                                                                                  
    image: ghcr.io/linuxserver/sabnzbd                                                                                                                                                                                                      
    container_name: sabnzbd                                                                                                                                                                                                                 
    environment:                                                                                                                                                                                                                            
      - PUID=${PUID}                                                                                                                                                                                                                        
      - PGID=${PGID}                                                                                                                                                                                                                        
      - TZ=${TZ}                                                                                                                                                                                                                            
    volumes:                                                                                                                                                                                                                                
      - "${CONFIG_DIRECTORY}/sabnzbd:/config"                                                                                                                                                                                               
      - "${DATA_DIRECTORY}/downloads/completed:/downloads"                                                                                                                                                                                  
      - "${DATA_DIRECTORY}/downloads/incomplete:/incomplete-downloads"                                                                                                                                                                      
    network_mode: "service:gluetun"                                                                                                                                                                                                         
    depends_on:                                                                                                                                                                                                                             
      - gluetun                                                                                                                                                                                                                             
    #    ports:                                                                                                                                                                                                                             
    #      - 8080:8080                                                                                                                                                                                                                      
    restart: unless-stopped
```

**Comment by qdm12** [comment]
Please hold, I'll add that compression flag back in the latest image since it's required.

For the public IP information on it's obtained from https://ipinfo.io maybe it's inaccurate. Let me know if the IP address is not the expected one though, and I'll check.

**Comment by qdm12** [comment]
Fixed in `:latest` image with dc674014ff22e0fb73ac27629458930f5f5ef5d6
